mappings/net/minecraft/client/gui/menu/RedErrorScreen.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS cxs net/minecraft/client/gui/menu/ErrorScreen
+CLASS cxs net/minecraft/client/gui/menu/RedErrorScreen
```

Comment:
why? what other error screen is there?

Reply:
The regular one. This one is the one with the red background: https://bugs.mojang.com/secure/attachment/122747/English.png

Reply:
Ah, I'd always thought that was it showing a red-tinted world skybox with no terrain loaded. Is it ever used for something other than the "Unable to load worlds" error screen?

Reply:
No, but it's not hardcoded for that error so its name should stay generic in case vanilla or a mod uses it for something else in the future. It could be called SevereErrorScreen if you want a more semantic name.

Reply:
I think `ServerErrorScreen` would be a lot better. Describing something by its visual features doesn't really feel right to me.

Reply:
`SevereErrorScreen` sounds preferable in my opinion.

Reply:
@Boundarybreaker "severe" (as in, all your worlds are gone), not "server".

Reply:
ahh, misread. Yeah that sounds great too.